pronto-rubocop.gemspec

```diff
@@ -33,8 +33,11 @@ Gem::Specification.new do |s|
   s.require_paths = ['lib']
 
   s.add_runtime_dependency('pronto', '~> 0.11.0')
-  s.add_runtime_dependency('rubocop', '>= 0.49.1', '< 1.0')
+  s.add_runtime_dependency('rubocop', '>= 0.49.1', '< 2.0')
+  s.add_development_dependency('appraisal')
   s.add_development_dependency('rake', '~> 12.0')
   s.add_development_dependency('rspec', '~> 3.4')
   s.add_development_dependency('rspec-its', '~> 1.2')
+  s.add_runtime_dependency('rubocop-rake')
+  s.add_runtime_dependency('rubocop-rspec')
```

Comment:
Why are these runtime dependencies needed now?

Reply:
Removed. rubocop 1.x recommended to add these gems.